c2c/template.py

```diff
@@ -348,6 +351,10 @@ def read_vars(vars_file):
         used['environment'] = []
     if 'runtime_environment' not in used:
         used['runtime_environment'] = []
+    if 'runtime_interpreted' not in used:
+        used['runtime_interpreted'] = {}
+    if 'runtime_postprocess' not in used:
+        used['runtime_postprocess'] = []
```

Comment:
This if+set can be replaced by: `used.setdefault('runtime_postprocess', [])`

Reply:
Thanks :-)